sway-ir/sway-ir-macros/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "sway-ir-macros"
```

Comment:
What is the reasoning for why this needs to be in a separate crate?

Reply:
Rust needs to build procedural macros before they can be run. This (currently) requires that procedural macros are defined in a separate crate to the crate which uses them so that the macro crate can be built first.

Reply:
Gotcha.